amical/data_processing.py

```diff
@@ -279,11 +279,11 @@ def sky_correction(imA, r1=None, dr=None, verbose=False, *, center=None, mask=No
             "Background not computed, likely because specified radius is out of bounds",
             RuntimeWarning,
         )
-    elif verbose:
-        print(
-            f"Sky correction of {backgroundB} was subtracted,"
-            f" remaining background is {backgroundC}."
-        )
+    # elif verbose:
+    #     print(
+    #         f"Sky correction of {backgroundB} was subtracted,"
+    #         f" remaining background is {backgroundC}."
+    #     )
```

Comment:
```suggestion
```

Reply:
I just remove this comment to have a clean progress bar if used with verbose. But actually, how can you deal with tqdm when you have such print between iteration?

Reply:
I don't know if that's possible. That's probably something you could do with `rich.progress` but that would be whole new dependency and migration would be a much bigger undertaking